Snakefile-sorting-hat

```diff
@@ -165,12 +165,15 @@ rule produce_recalculations_call:
         mkdir -p {wildcards.accession}/logs
         exec &> {log}
         pushd {input.working_directory}
-        snakemake {config['sm_options']} --config \
+
+
+        snakemake --latency-wait 100 {config['sm_options']} --config \
             accession={wildcards.accession} \
             tool={get_tool(wildcards.accession)} \
             metadata_summary=../{input.metadata} \
             bioentities_properties={params.bioentities_properties} \
             delete_previous_output={params.delete_previous_output} \
+            skip_steps_file={params.skip_steps_file} \
```

Comment:
Could you remind me what was this about? was it to skip steps for datasets where we don't have raw sequences?

Reply:
This is regarding a non-public YAML file with a reduce list of experiments that specifically can skip certain steps. This is necessary in absence of certain files.